core/tests/puppeteer-acceptance-tests/spec/blog-admin-tests/assign-roles-to-users-and-change-tag-properties.spec.ts

```diff
@@ -55,34 +55,46 @@ describe('Blog Admin', function () {
       );
 
       await superAdmin.expectUserNotToHaveRole('guestUsr1', ROLES.BLOG_ADMIN);
+      await superAdmin.screenshotMatch('expectUserNotToHaveRole');
```

Comment:
I don't think this is a superAdmin thing ... it's more of a utility function or a general utility function I suppose. Can we do something like `await screenshotUtils.expectScreenshotToMatch(...)`?